package.json

```diff
@@ -89,6 +89,7 @@
     "base64-loader": "^1.0.0",
     "clean-webpack-plugin": "^4.0.0",
     "copy-webpack-plugin": "^11.0.0",
+    "cross-env": "^7.0.3",
```

Comment:
Do we actually need `cross-env` for this? I thought it would just pick it up

Reply:
This is different about setting the ENV in the CMD each platform (window need "export", linux need "set",...), so I think It needs this library to make it simple, no worry, it is lightweight

Reply:
Oh we need to export the variable, no worries